### PR TITLE
Fix a few resharper warnings and appy a few its suggestions

### PR DESCRIPTION
**Pull request by goremykin**
**Changes**
This PR removes some minor redundancy and applies some resharper's suggestions (mostly modern sugar).
Commits are quite descriptive and split by suggestion / warning type, so I recommend review it commit by commit.

### CODE REVIEW

File: Emby.Server.Implementations/Channels/ChannelManager.cs [hunk 1]
```diff
@@ -1156,7 +1156,7 @@ private async Task<BaseItem> GetChannelItemEntityAsync(ChannelItemInfo info, ICh
 
                 if (info.People is not null && info.People.Count > 0)
                 {
-                    _libraryManager.UpdatePeople(item, info.People);
+                    await _libraryManager.UpdatePeopleAsync(item, info.People, cancellationToken).ConfigureAwait(false);
```
Comment (goremykin) [inline Emby.Server.Implementations/Channels/ChannelManager.cs:1159]: Btw, do we have a synchronization context somewhere?
I mean, do we really need `ConfigureAwait` everywhere?

File: Emby.Server.Implementations/HttpServer/WebSocketConnection.cs [hunk 2]
```diff
@@ -12,7 +12,6 @@
 using MediaBrowser.Controller.Net.WebSocketMessages;
 using MediaBrowser.Controller.Net.WebSocketMessages.Outbound;
 using MediaBrowser.Model.Session;
-using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.Logging;
 
 namespace Emby.Server.Implementations.HttpServer
```
Comment (goremykin) [inline Emby.Server.Implementations/HttpServer/WebSocketConnection.cs:17]: Are we planning switching to the file-scoped namespace everywhere?

File: Emby.Server.Implementations/ScheduledTasks/Tasks/ChapterImagesTask.cs [hunk 3]
```diff
@@ -115,7 +115,8 @@ public async Task ExecuteAsync(IProgress<double> progress, CancellationToken can
             {
                 try
                 {
-                    previouslyFailedImages = File.ReadAllText(failHistoryPath)
+                    var failHistoryText = await File.ReadAllTextAsync(failHistoryPath, cancellationToken).ConfigureAwait(false);
+                    previouslyFailedImages = failHistoryText
```
Comment (Bond-009) [inline Emby.Server.Implementations/ScheduledTasks/Tasks/ChapterImagesTask.cs:119]: Why in 2 parts?

**Review by Bond-009 — APPROVED** [review]
For the future I'd prefer it if you wouldn't make cosmetic only changes to the RSSDP project.
Fewer changes compared to upstream will make it easier to diff and hopefully eventually replace.

File: Emby.Server.Implementations/ScheduledTasks/Tasks/ChapterImagesTask.cs (hunk 3 above)
Comment (goremykin) [reply Emby.Server.Implementations/ScheduledTasks/Tasks/ChapterImagesTask.cs:119]: @Bond-009, it looks a bit cumbersome. Let's fix it if you don't think so
```
previouslyFailedImages = (await File.ReadAllTextAsync(failHistoryPath, cancellationToken).ConfigureAwait(false))
    .Split('|', StringSplitOptions.RemoveEmptyEntries)
    .ToList();
```

Comment (goremykin) [reply Emby.Server.Implementations/ScheduledTasks/Tasks/ChapterImagesTask.cs:119]: @Bond-009, and could you check my questions above?

File: Emby.Server.Implementations/Channels/ChannelManager.cs (hunk 1 above)
Comment (Bond-009) [reply Emby.Server.Implementations/Channels/ChannelManager.cs:1159]: No, but we should keep using them for the stuff that gets published to nuget at least

File: Emby.Server.Implementations/HttpServer/WebSocketConnection.cs (hunk 2 above)
Comment (Bond-009) [reply Emby.Server.Implementations/HttpServer/WebSocketConnection.cs:17]: Maybe, but I wouldn't go and change them as that would mess with git blame

File: Emby.Server.Implementations/ScheduledTasks/Tasks/ChapterImagesTask.cs (hunk 3 above)
Comment (Bond-009) [reply Emby.Server.Implementations/ScheduledTasks/Tasks/ChapterImagesTask.cs:119]: I didn't notice the split below, I'd move that to the same line

File: Emby.Server.Implementations/HttpServer/WebSocketConnection.cs (hunk 2 above)
Comment (goremykin) [reply Emby.Server.Implementations/HttpServer/WebSocketConnection.cs:17]: @Bond-009, At least in jetbrains rider it doesn't affect git blame at all. But I am not sure about VS and git cli.

**Review by Shadowghost — APPROVED** [review]
Looks fine but keep in mind that merging this will likely break a lot of existing PRs

**Pull request merged**